docs/resources/control-flow/user-interactivity/forms/form-widgets/dropdown.md

```diff
@@ -0,0 +1,310 @@
+
+:::info
+To clear the selection, you can use the [Reset Form Fields](#) action and choose the **Reset Dropdown Fields** option. Then, simply select the name of the dropdown widget you wish to reset.
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/da8f8e2b0ceb4509a2d69dd25e2d94e2?sid=2df82d07-a178-495c-ba7c-ec2ed059c649"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+
+### Changing dropdown size
+
+To change the height and width of the dropdown, select the **DropDown** widget, move to the **Properties Panel > DropDown Properties > enter the Width and Height value**.
+
+### Set max height
+
+If needed, you can also control the dropdown height using the **Max Height** property.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/06fb6dfef8304bb5b3a2d1be4e8fe6b0?sid=80820c77-9ea4-4024-8af2-df09ce978f5c"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Adding margin
+
+Margin adds a space between the DropDown's text and its border. To change the margin, select the **DropDown** widget, move to the **Properties Panel > DropDown Properties >** find the **Margin** property, and change the values.
+
+![Adding margin](../../imgs/adding-margin.gif)
+
+### Changing background color
+
+To change the background color, move to the **Properties Panel > DropDown Style > set the Fill Color**.
+
+![Changing background color](../../imgs/changing-background-color.png)
+
+### Changing menu elevation
+
+Menu elevation adds a shadow to the dropdown, giving it a sense of depth and making it appear above the surface it is placed on.
+
+To change the menu elevation (depth or Z-axis), move the **Properties Panel >** enter the **Menu** **Elevation** value.
+
+:::info
+The higher value draws the bigger size of the shadow.
+:::
+
+### Adding border
+
+See how to [add a border](#).
```

Comment:
Not sure if we have a common page for this, we can remove this section otherwise.

Reply:
Added instructions to the common page and linked from here.